src/docfx/config/ops/OpsPreProcessor.cs

```diff
@@ -67,7 +69,7 @@ private void PreProcessMonoDocXml()
                             ? null
                             : Path.GetFullPath(Path.Combine(_buildOptions.DocsetPath, ".fallback", monodocConfig.OutputYamlFolder));
                         var (repository, _) = _repositoryProvider.GetRepository(new PathString(xmlDirectory));
-                        ECMA2YamlConverter.Run(
+                        result &= ECMA2YamlConverter.Run(
```

Comment:
return false on first failure? There doesn't seem to be value in processing more entries in case of failure.

Reply:
The failure is most likely caused by user content, so I think it would be worth to report all content errors.